tools/build_node.js

```diff
@@ -173,8 +173,9 @@ async function buildNode(options) {
     if (stat.isDirectory()) return;
 
     if (file.endsWith(".css")) {
-      installCleanCSS(`./src/styles/${file}`, `styles/${file}`);
-      installCleanCSS(`./src/styles/${file}`, `scss/${file.replace(".css", ".scss")}`);
+      installCleanCSS(`./src/styles/${file}`, `styles/${file}`, false);
+      installCleanCSS(`./src/styles/${file}`, `styles/${file.replace(".css", ".min.css")}`, true);
```

Comment:
I'm confused, isn't passing false the same as not passing the argument?  Both `false` and `undefined` (which not passed) are falsy values...

Reply:
The first false/undefined is correct. About the second point - please amend my PR directly to wanted style/function.